doc/admin.md

```diff
@@ -0,0 +1,565 @@
+# Administration
+
+-   [Configuration](#configuration)
+    -   [Configuration parameters:](#configuration-parameters)
+        -   [loglevel](#loglevel)
+        -   [contextBroker](#contextbroker)
+        -   [server](#server)
+        -   [stats](#stats)
+        -   [authentication](#authentication)
+        -   [deviceRegistry](#deviceregistry)
+        -   [mongodb](#mongodb)
+        -   [iotManager](#iotmanager)
+        -   [types](#types)
+        -   [eventType](#eventtype)
+        -   [service](#service)
+        -   [subservice](#subservice)
+        -   [providerUrl](#providerurl)
+        -   [iotaVersion](#iotaversion)
+        -   [appendMode](#appendmode)
+        -   [dieOnUnexpectedError](#dieonunexpectederror)
+        -   [singleConfigurationMode](#singleconfigurationmode)
+        -   [timestamp](#timestamp)
+        -   [defaultResource](#defaultresource)
+        -   [defaultKey](#defaultkey)
+        -   [componentName](#componentname)
+        -   [pollingExpiration](#pollingexpiration)
+        -   [pollingDaemonFrequency](#pollingdaemonfrequency)
+        -   [autocast](#autocast)
+        -   [multiCore](#multicore)
+        -   [fallbackTenant](#fallbacktenant)
+        -   [fallbackPath](#fallbackpath)
+        -   [explicitAttrs](#explicitattrs)
+        -   [defaultEntityNameConjunction](#defaultentitynameconjunction)
+        -   [relaxTemplateValidation](#relaxtemplatevalidation)
+    -   [Configuration using environment variables](#configuration-using-environment-variables)
+-   [Logs](#logs)
+    -   [Errors](#errors)
+-   [Alarms](#alarms)
+
+The **IoT Agent node library** is not a standalone product and should be added as a dependency to `package.json` of the
+project is going to be used in. The library is published in the NPM repository, so it can be added to the project by
+adding the following line to the `package.json` file:
+
+```json
+...
+"dependencies": {
+    "iotagent-node-lib": "*",
```

Comment:
This is a kind o "blind" dependency (meaning "the last version of the library, whatever it is") and I think it is not the recomended way...

What about?

```suggestion
    "iotagent-node-lib": "x.y.z",
```

explaning just below that `x.y.z` means an actual library version.